src/types.ts

```diff
@@ -18,6 +18,13 @@ export type State<T> = Readonly<{
   }>;
 }>;
 
+export interface RequestOptions {
```

Comment:
one final nit: can you export this from teh package itself (root index.ts)

Reply:
Done